src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementForm.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useCallback } from 'react';
+import { defineMessages } from 'react-intl';
+import { ColonyRole, ROOT_DOMAIN_ID } from '@colony/colony-js';
+import sortBy from 'lodash/sortBy';
+
+import { Address } from '~types/index';
+import { DomainFieldsFragment } from '~data/generated';
+import { InputLabel, Textarea, Select } from '~core/Fields';
+
+import PermissionManagementCheckbox from './PermissionManagementCheckbox';
+import { availableRoles } from './constants';
```

Comment:
Why also import the `availableRoles` if you already have `ColonyRole` imported? 

They're the same thing.

Reply:
Never mind about this, we discussed it privately and it can stay like it is.